src/SDKs/IotHub/IotHub.Tests/Properties/AssemblyInfo.cs

```diff
@@ -35,5 +35,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.1")]
-[assembly: AssemblyFileVersion("1.0.0.1")]
+[assembly: AssemblyVersion("1.0.0.2")]
```

Comment:
@anusapan AssemblyVersion only changes when you update the Major version. Please do not update AssemblyVersion and make it same as AssemblyFileVersion

Reply:
reverted